package.json

```diff
@@ -5,6 +5,7 @@
   "repository": "git@github.com:plone/plone.staticresources.git",
   "license": "BSD-3-Clause",
   "dependencies": {
+    "bootstrap-icons": "",
```

Comment:
We should fix it to a version (range). Something like "^4.0.3" or whatever.

Reply:
bootstrap-icons are in alpha right now, don't think it would be a problem to always get the newest, at least for now

Reply:
i'm ok with that.